testing/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# based on https://github.com/AFLplusplus/AFLplusplus/blob/stable/Dockerfile
+
+FROM ubuntu:20.04 AS toxcore-fuzz
```

Comment:
Why do you name it?

Reply:
To be able to easily reference it. Is this not how it's done in Docker?

Reply:
Yeah, but where are you referencing it?

Reply:
Just when using it in interactive command lines

Reply:
Does that actually work? Can you show a sequence of command line operations you'd do that make use of this name?